src/main/scala/chiseltest/stage/ChiselTestCli.scala

```diff
@@ -0,0 +1,14 @@
+package chiseltest.stage
+
+import firrtl.options.Shell
+
+trait ChiselTestCli {
+  this: Shell =>
+  parser.note("ChiselTest Options")
+  Seq(
+    TreadleBackendAnnotation,
+    VerilatorBackendAnnotation,
+    VcsBackendAnnotation,
+    WriteVcdAnnotation
```

Comment:
Do you want to include the `TestFunctionAnnotation` here, too?

Reply:
This has a issue:
`TestFunctionAnnotation` method serialization.
Hot to pass the method in string and call with reflection for this still remains unimplemented.
I think it can be implemented in the future PRs.